spotty/spotty/Data/Network/Network.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Network.swift
+//  spotty
+//
+//  Created by Ivaylo Kalaydzhiev on 10.06.22.
+//
+
+import Foundation
+
+struct Network {
+    
+    /// This method performs a ``URLRequest`` with the supplied ``URLSession`` and then parses the retrieved ``Data``,
+    ///  presuming it is in JSON format into the type, specified by the  completion handler
+    /// - Parameters:
+    ///    - urlRequest: The ``URLRequest`` to be performed.
+    ///    - urlSession: A ``URLSession`` on which to perform the ``URLRequest``.
+    ///    - completion: A block of code that handles both successfull data retrieval and parsing and potential errors.
+    static func performRequest<T: Decodable>(urlRequest: URLRequest,
+                                             urlSession: URLSession = URLSession.shared,
+                                             completion: @escaping (Result<T, Error>) -> Void) {
+        
+        urlSession.dataTask(with: urlRequest) { data, response, error in
+            if let data = data {
+                
+                // Delete those two lines
```

Comment:
Remove comments

Reply:
I want to keep those lines for testing purposes. I will delete them at a later point.

Reply:
In such a cases you can mention something like
_// Comments for testing purposes
// TODO: - Delete before release_